src/ontodia/diagram/model.ts

```diff
@@ -470,8 +470,14 @@ function placeholderTemplateFromIri(iri: string): ElementModel {
 export function uri2name(uri: string): string {
     const hashIndex = uri.lastIndexOf('#');
     if (hashIndex !== -1 && hashIndex !== uri.length - 1) {
-        return uri.substring(hashIndex + 1);
+        const textAfterHash = uri.substring(hashIndex + 1);
+        if (textAfterHash.length > 0) {
```

Comment:
It's impossible for `textAfterHash` to be empty (because otherwise earlier check `hashIndex !== uri.length - 1` would be false).